docs/library-specification.md

```diff
@@ -72,7 +72,7 @@ Each folder has a specific purpose (sources, examples, documentation, etc). Fold
 
 #### Library Root folder
 
-The library root folder name must start with a basic letter (`A`-`Z` or `a`-`z`) or number (`0`-`9`), followed by basic letters, numbers, spaces (` `), underscores (`_`), dots (`.`) and dashes (`-`). The maximum length is 63 characters.
+The library root folder name must start with a basic letter (`A`-`Z` or `a`-`z`) or number (`0`-`9`), followed by basic letters, numbers, spaces (``), underscores (`_`), dots (`.`) and dashes (`-`). The maximum length is 63 characters.
```

Comment:
Ouch, that's not good. We could exclude the whole line for formatting to solve this, I want to see if there's another solution though.

Reply:
I've noticed that writing `( )` instead of  ```(` `)``` doesn't remove the space between parenthes, it's not a fixed size space though. 

I can either do this or ignore formatting for that line.